packages/runtime/container-runtime/src/pendingStateManager.ts

```diff
@@ -375,6 +400,11 @@ export class PendingStateManager implements IDisposable {
 				const localOpMetadata = await this.stateHandler.applyStashedOp(nextMessage.content);
 				if (this.stateHandler.isAttached()) {
 					nextMessage.localOpMetadata = localOpMetadata;
+					// NOTE: This runtimeOp has been roundtripped through string, which is technically lossy,
+					// e.g. handles will be RemoteFluidObjectHandles instead of a handle backed directly by the target object
```

Comment:
where do handles become RemoteFluidObjectHandles  as Json.parse won't do it automatically

Reply:
Oh, my comment is very misleading though I'll fix that